flashlight/meter/PerformanceAudit.h

```diff
@@ -0,0 +1,347 @@
+/*
```

Comment:
As above - you're planning on removing this after you're done benchmarking, yes?